stock_request_submit/models/stock_request.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Open Source Integrators
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models
+
+REQUEST_STATES = [
```

Comment:
You could add REQUEST_STATES on a model function in order to allow inheritance. What do you think?

Reply:
From stock_request?

Reply:
for example, and will be inherited by stock.request.order and you could add the submit state on `stock_request_submit` easily.

Reply:
I should call the method and send all states to get submitted state the second. Maybe for v13 :smile_cat:

Reply:
I don't understand. Why the order of the strings in the selection is important?

Reply:
It defines the sequence of states in the status bar.